tests/test_vcs_flake8.py

```diff
@@ -22,7 +22,7 @@ def testFlake8(self):
         print()
         print()
         print()
-        P = subprocess.Popen(shlex.split("flake8 --show-source --statistics --ignore=F999,F405,E121,E123,E126,E226,E24,E704 --max-line-length=120 %s" % pth),
+        P = subprocess.Popen(shlex.split("flake8 --show-source --statistics --ignore=F999,F405,E121,E123,E126,E226,E24,E704,W504 --max-line-length=120 %s" % pth),
```

Comment:
Now I see what you meant about those `W504`, as even though the flake8 test passed on my machine last night, I noticed a bunch of those coming from my branch on CircleCI this morning.

Reply:
Yes I have no idea why they show up on circleci though... I the exact same version here and the warnings do not show up... Mayve a python difference?

Reply:
@scottwittenburg if you're ok with the PR please approve I'll merge in master and we can merge back into your branch.

Reply:
@scottwittenburg pushed the change